proteus/Geom.py

```diff
@@ -0,0 +1,228 @@
+"""
```

Comment:
All this functionality should be moved  into MeshTools since these are all functions for operating on mesh elements and meshes. We also have an Euclidean Geometry module (EGeometry) for basic geometric  calculations, but the  functions in here are specific  to meshes.